src/CTA.FeatureDetection.ProjectType/CompiledFeatures/CoreWCFServiceCodeFeature.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using Codelyzer.Analysis;
+using CTA.FeatureDetection.Common.Models.Features.Base;
+using CTA.FeatureDetection.Common.Models.WCF;
+using CTA.FeatureDetection.Common.WCFConfigUtils;
+
+namespace CTA.FeatureDetection.ProjectType.CompiledFeatures
```

Comment:
Add another feature to determine if ServiceHost is presence.... the way to determine that is Either there must a be file which ends with .svc or in the code there must ServiceHost declaration.... let me know if it consumes lots of compute time...